docs/antrea-network-policy.md

```diff
@@ -1102,6 +1107,46 @@ spec:
       - fqdn: "svcA.default.svc.cluster.local"
 ```
 
+## Node Selector
+
+NodeSelector selects certain Nodes which match the label selector.
+When used in the `to` field of an egress rule, it adds the Node IPs to the rule's destination address group;
+when used in the `from` field of an ingress rule, it adds the Node IPs to the rule's source address group.
+
+Notice that when a rule with a nodeSelector applies to a Node, it only restricts the traffic to/from certain IPs of the Node.
+The IPs include:  
+
+1. The Node IP (the IP address in the Node API object)
+2. The Antrea gateway IP (the IP address of the interface antrea-agent will create and use for Node-to-Pod communication)
+3. The transport IP (the IP address of the interface used for tunneling or routing the traffic across Nodes) if it's different from Node IP
+
+Traffic to/from other IPs of the Node will be ignored.
```

Comment:
```suggestion
Traffic to/from other IPs of the Node will be ignored.
Meanwhile, `NodeSelector` doesn’t affect the traffic between local Pod and local Node, the traffic to/from local Node will be bypassed by default, this makes sure that [agents on a node (e.g. system daemons, kubelet) can communicate with all pods on that node](https://kubernetes.io/docs/concepts/cluster-administration/networking/#the-kubernetes-network-model).For more information, see [https://github.com/antrea-io/antrea/issues/280](https://github.com/antrea-io/antrea/issues/280)
```

Reply:
@hongliangl @tnqn Any comments about the new doc change?

Reply:
pod -> Pod 
node -> Node

Reply:
Good catch, updated.